doc/source/turn-off.rst

```diff
@@ -7,18 +7,34 @@ When you are done with your hub, you should delete it so you are no longer
 paying money for it. The following sections describe how to delete your
 JupyterHub resources on various cloud providers.
```

Comment:
Could we add one sentence that describes a short high-level overview of what we'll do in the page? E.g.:

```
Tearing down your JupyterHub entails:

1. Deleting your Kubernetes namespace, which deletes all objects created and managed by Kubernetes
2. Deleting any computational resources you've requested from the cloud provider
3. Running a final check to make sure there aren't any lingering resources that haven't been deleted (e.g., storage volumes in some cloud providers)
```

Reply:
Great suggestion @choldgraf. This would be helpful at the start of the section 👍